
# HG changeset patch
# User shade
# Date 1544788617 -3600
# Node ID 33d33996a638f3eb44befe3c21fa8788808dc3e8
# Parent  c74f074c532a2df83f733e6aa4982187421e0dc3
8215353: x86_32 build failures after JDK-8214751 (X86: Support for VNNI Instructions)
Reviewed-by: thartmann, kvn

diff -r c74f074c532a -r 33d33996a638 src/hotspot/cpu/x86/x86_32.ad
--- a/src/hotspot/cpu/x86/x86_32.ad	Fri Dec 14 12:02:39 2018 +0100
+++ b/src/hotspot/cpu/x86/x86_32.ad	Fri Dec 14 12:56:57 2018 +0100
@@ -7760,9 +7760,9 @@
   match(Set dst (MulAddS2I (Binary dst src1) (Binary src2 src3)));
   effect(KILL cr, KILL src2);
 
-  expand %{ mulI_rReg(dst, src1, cr);
-           mulI_rReg(src2, src3, cr);
-           addI_rReg(dst, src2, cr); %}
+  expand %{ mulI_eReg(dst, src1, cr);
+           mulI_eReg(src2, src3, cr);
+           addI_eReg(dst, src2, cr); %}
 %}
 
 // Multiply Register Int to Long

